### DISCUSSION TITLE
Reason for CyclicTuples

### DISCUSSION THREAD

**Discussion by jgarte** [Abjad/abjad#1487] [Q&A]
@trevorbaca 

How have you used [CyclicTuples](https://github.com/Abjad/abjad/blob/810dbaacd00b6347b8d4983c5fc01555bc8a39db/abjad/cyclictuple.py#L9) in the past? 

To solve what types of problems? 

In what contexts have you used them?

**Comment by trevorbaca** [comment]
Any time you want to read values in a loop.
